daemon/create.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // ContainerCreate takes configs and creates a container.
-func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hostConfig *runconfig.HostConfig, adjustCPUShares bool) (types.ContainerCreateResponse, error) {
+func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hostConfig *runconfig.HostConfig, adjustCPUShares bool, fromBuilder bool) (types.ContainerCreateResponse, error) {
```

Comment:
Got it. I tried with a private function/wrappers, but it looks (IMO) way better with a struct. Updated.